generated/nidaqmx/nidaqmx_service.h

```diff
@@ -406,9 +410,22 @@ class NiDAQmxService final : public NiDAQmx::WithCallbackMethod_RegisterSignalEv
   ::grpc::Status WriteRaw(::grpc::ServerContext* context, const WriteRawRequest* request, WriteRawResponse* response) override;
   ::grpc::Status WriteToTEDSFromArray(::grpc::ServerContext* context, const WriteToTEDSFromArrayRequest* request, WriteToTEDSFromArrayResponse* response) override;
   ::grpc::Status WriteToTEDSFromFile(::grpc::ServerContext* context, const WriteToTEDSFromFileRequest* request, WriteToTEDSFromFileResponse* response) override;
+
+  bool is_enabled();
 private:
   NiDAQmxLibraryInterface* library_;
   ResourceRepositorySharedPtr session_repository_;
+
+  struct NiDAQmxFeatureToggles
+  {
+    using CodeReadiness = nidevice_grpc::FeatureToggles::CodeReadiness;
+    NiDAQmxFeatureToggles(const nidevice_grpc::FeatureToggles& feature_toggles);
+
+    bool is_enabled;
+    bool is_allow_undefined_attributes_enabled;
```

Comment:
Note: This will be the toggle for accessing undefined raw attributes. I haven't implemented that logic yet.